addons/xterm-addon-webgl/src/renderLayer/CursorRenderLayer.ts

```diff
@@ -54,6 +54,14 @@ export class CursorRenderLayer extends BaseRenderLayer {
     this.onOptionsChanged(terminal);
   }
 
+  public override dispose(): void {
+    if (this._cursorBlinkStateManager) {
+      this._cursorBlinkStateManager.dispose();
+      this._cursorBlinkStateManager = undefined;
+    }
```

Comment:
Little more concise:

```suggestion
    this._cursorBlinkStateManager?.dispose();
    this._cursorBlinkStateManager = undefined;
```